client/flutter/lib/pages/symptom_checker/symptom_checker_view.dart

```diff
@@ -133,15 +154,17 @@ class _SymptomCheckerViewState extends State<SymptomCheckerView>
     _model.answerQuestion(answerIds);
     if (!_model.isFatalError && _model.results != null) {
       Navigator.of(context)
-          .pushNamed('/symptom-checker-results', arguments: _model);
+          .pushReplacementNamed('/symptom-checker-results', arguments: _model);
```

Comment:
I had it this way before, but we want the back on results to go to the last Q.

Reply:
@advayDev1  The only way to escape would then be to go back through every single question.

Reply:
No.  You press the 'Done' button?

Reply:
I am approving and we can fix this in the next PR.